app/models/notification.rb

```diff
@@ -172,4 +174,12 @@ def subject_number
   def display_thread?
     Octobox.include_comments? && subjectable? && subject.present? && user.display_comments?
   end
+
+  def github_client
```

Comment:
I'd probably move this method into `lib/octobox/notifications/sync_subject.rb` as that looks like it's the only place it gets used